### ISSUE TITLE
linter: `--report-unused-disable-directives` does not work for multiple rules

### ISSUE THREAD

**Issue by Sysix** [oxc-project/oxc#11709]
### What version of Oxlint are you using?

1.11.0

### What command did you run?

`oxlint --report-unused-disable-directives`

### What does your `.oxlintrc.json` config file look like?

```jsonc
None
```

### What happened?

oxlint should report unused-disable-directives with sub spans for rules.

Example:
```
// oxlint-disable-next-line no-console no-debugger -- no console should be reported
(() => { debugger; })();
```

Would be great because #11708 is implementing a fix for it.

**Comment by Sysix** [comment]
False report: it is comma separated
